Test_JSONFiles_MSG/ta.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Test files should be added here: https://github.com/cdisc-org/cdisc-rules-engine/tree/main/tests/resources
Also we should add unit tests for the dataset json reader

Reply:
Can you give push access so that I can create a folder and add Dataset-JSON files to it?
Many thanks in advance,
Jozef